src/Raven.Server/Rachis/RachisConsensus.cs

```diff
@@ -1724,7 +1733,8 @@ public string HardResetToNewCluster(string nodeTag = "A")
                     },
                     new Dictionary<string, string>(),
                     new Dictionary<string, string>(),
-                    lastNode
+                    lastNode,
+                    -1
```

Comment:
I think we do need to notify the client about cluster topology change. So we probably should set it to a higher value here.

Reply:
done